eng/common/scripts/get-codeowners.ps1

```diff
@@ -13,34 +16,89 @@ if (!(Test-Path $codeOwnersLocation)) {
 }
 
 $codeOwnersContent = Get-Content $codeOwnersLocation
+$previousLine
 
 foreach ($contentLine in $codeOwnersContent) {
   if (-not $contentLine.StartsWith("#") -and $contentLine){
     $splitLine = $contentLine -split "\s+"
     
     # CODEOWNERS file can also have labels present after the owner aliases
     # gh aliases start with @ in codeowners. don't pass on to API calls
-    $ownedFolders[$splitLine[0].ToLower().Trim("/")] = ($splitLine[1..$($splitLine.Length)] `
-      | ? { $_.StartsWith("@") } `
-      | % { return $_.substring(1) }) -join ","
+
+    $aliases = ($splitLine[1..$($splitLine.Length)] | ? { $_.StartsWith("@") } | % { return $_.substring(1) }) -join ","
+    $labels = ""
+
+    if ($null -ne $previousLine -and $previousLine.StartsWith("# PRLabel:"))
```

Comment:
This assumes exact spacing between `#` and PRlabel we should be a little more flexible. Make sure you have a look at https://github.com/Azure/azure-sdk-tools/blob/f9a1eef96d4b878a60e0dfc687b6f5067ccc2562/tools/CreateRuleFabricBot/CreateRuleFabricBot/Rules/PullRequestLabel/CodeOwnerEntry.cs#L51 and https://github.com/Azure/azure-sdk-tools/blob/11713539bb97ccb0342db98c7c400d31233a42d5/tools/CreateRuleFabricBot/CreateRuleFabricBot/Rules/PullRequestLabel/PullRequestLabelFolderCapability.cs#L159 and try to mimic the same behavior in PS.

Reply:
I updated the Logic.